htmd/parameterization/ff.py

```diff
@@ -202,16 +202,11 @@ def writeFrcmod(self, rtf, filename):
                 if len(prm) == 0:
                     prm.append(prmx[0])
 
-                for pi in range(len(prm)):
-                    p = prm[pi]
-                    sign = 1
+                for p in prm:
                     scee = 1. / p.e14
                     scnb = 2.
-                    if pi < (len(prm) - 1):
-                        sign = -1
                     print("%2s-%2s-%2s-%2s 1 %12.6f %12.6f %12.6f %12.6f %12.6f" %
-                          (t[i.types[0]], t[i.types[1]], t[i.types[2]], t[i.types[3]], p.k0, p.phi0, sign * p.n, scee,
-                           scnb),
+                          (t[i.types[0]], t[i.types[1]], t[i.types[2]], t[i.types[3]], p.k0, p.phi0, p.n, scee, scnb),
```

Comment:
What happened to the sign? No longer needed?

Reply:
I have no idea, why it was needed at the first place.